ports/servo/browser.rs

```diff
@@ -315,10 +323,13 @@ impl Browser {
                     self.loading_state = Some(LoadingState::Loaded);
                 }
                 EmbedderMsg::CloseBrowser => {
-                    self.browser_id = None;
-                    // Nothing left to do for now,
-                    // but could hide a tab, and show another one, instead of quitting.
-                    self.event_queue.push(WindowEvent::Quit);
+                    // TODO: close the appropriate "tab".
```

Comment:
I'm not sure what this todo is about.

Reply:
Current we just pop from the list of browsers(and quit constellation when the list is empty). To properly implement tabs one day, we probably should close the browser from which the `CloseBrowser` originated.